src/plugins/okta/login.ts

```diff
@@ -90,9 +110,27 @@ export const oktaLogin = async (org: OrgData) =>
     })
   );
 
-/** Retrieves a SAML response for an okta app */
+/**
+ * Converts OIDC tokens into a SAML assertion for AWS federated authentication.
+ *
+ * This function bridges the gap between modern OIDC authentication (used by P0 CLI)
+ * and legacy SAML federation (required by AWS IAM). It performs a two-step process:
+ *
+ * 1. **Token Exchange (OIDC → Web SSO Token)**:
```

Comment:
The term `Web SSO Token` is pretty generic sounding - is this an Okta App Token? I imagine we're doing token exchange w/ Okta based off a configured Okta App, but this language seems more universal than that